xbmc/windowing/osx/WinEventsOSXImpl.mm

```diff
@@ -90,19 +93,19 @@ - (unichar)OsxKey2XbmcKey:(unichar)character
 {
   switch (character)
   {
-    case 0x1c:
-    case 0xf702:
+    case 0x1c: // kVK_ANSI_8
```

Comment:
import `Events.h` from Carbon framework, the constants are there. https://stackoverflow.com/a/4642095/1971301